creator-node/src/snapbackSM.js

```diff
@@ -1,10 +1,14 @@
 const config = require('./config')
 const Bull = require('bull')
+const axios = require('axios')
 const utils = require('./utils')
+const models = require('./models')
 const { logger } = require('./logging')
 
 const DevDelayInMS = 10000
 
+// TODO: Discuss w/draj how to handle a long upload where the stateMachine starts processing during the operation
```

Comment:
Discuss with @dmanjunath  how to handle a long upload where the stateMachine starts processing during the operation

Reply:
post-discussion we decided to intentionally not special-case this and allow an additional sync to potentially be issued given the existing lock